pkg/operator/logging/logging.go

```diff
@@ -51,6 +43,14 @@ func DefaultZapConfig(ctx context.Context, component string) zap.Config {
 		// Webhooks are deprecated, so support for changing their log level is also deprecated
 		logLevel = lo.Must(zap.ParseAtomicLevel(l))
 	}
+	outputPaths := []string{"stdout"}
```

Comment:
Rather than the this being a default here, should we just rely on the options and their defaults?